### ISSUE TITLE
Make attribute-map.yml path customizable

### ISSUE THREAD

**Issue by morissetcl** [apokalipto/devise_saml_authenticatable#236]
Hello,

I am wondering if it will be possible to make `attribute-map.yml` path customizable? It can happens that we already use this (generic) naming in a project, we can also want to be more explicit in the naming(`saml-attribute-map.yml`) or put this file under a module.

What do you think? 
If it make sense I will be happy to help.

Thanks.

**Comment by adamstegman** [comment]
Yeah, I could see this being a sensible option. You could also solve this by setting `saml_attribute_map_resolver` to your own class and reading from a different file.
